### PR TITLE
[stable/goldilocks]: Add `controller.rbac.extraClusterRoleBindings`

### PR DESCRIPTION
**Pull request by ivanfetch-wt**
**Why This PR?**
Add the ability to specify additional ClusterRoleBindings for the Goldilocks cluster role. This helps grant access to resources, E.G. via the built-in `view` ClusterRole.

**Checklist:**

* [x] I have included the name of the chart in the title of this PR in square brackets i.e. `[stable/goldilocks]`.
* [x] I have updated the chart version in `Chart.yaml` following Semantic Versioning.
* [x] Any new values are backwards compatible and/or have sensible default.
* [x] Any new values have been added to the README for the Chart, or `helm-docs --sort-values-order=file` has been run for the charts that support it.

### CODE REVIEW

File: stable/goldilocks/templates/controller-clusterrolebinding.yaml [hunk 1]
```diff
@@ -17,4 +17,27 @@ subjects:
   - kind: ServiceAccount
     name: {{ include "goldilocks.fullname" . }}-controller
     namespace: {{ .Release.Namespace }}
+
+    {{- range $.Values.controller.rbac.extraClusterRoleBindings }}
```
Comment (sudermanjr) [inline stable/goldilocks/templates/controller-clusterrolebinding.yaml]: nit: 
```suggestion
{{- range $.Values.controller.rbac.extraClusterRoleBindings }}
```

**Review by sudermanjr — CHANGES_REQUESTED** [review]
Can you please add an extraClusterRoleBinding to the test file in ci/test-values.yaml? This will ensure that we cover it with End-to-end tests.

File: stable/goldilocks/ci/test-values.yaml [hunk 2]
```diff
@@ -31,6 +31,8 @@ controller:
 dashboard:
   basePath: goldilocks
   ingress:
+    extraClusterRoleBindings:
+      - view
```
Comment (sudermanjr) [inline stable/goldilocks/ci/test-values.yaml]: I don't think that's the right place for this. Shouldn't this go under `controller.rbac`?

**Review by sudermanjr — CHANGES_REQUESTED** [review]

**Review by rbren — APPROVED** [review]

**Review by sudermanjr — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**